index.html

```diff
@@ -148,8 +153,8 @@ <h1 color="#FFFFFF" class="css-1rvqtgb">City of Boulder Careers</h1>
                                     </div>
                                 </div>
                                 <div class="css-1qm1lh"></div>
-                                <fieldset class="css-1dw6sgr">
-                                    <legend class="css-i19yjz">Filters</legend>
+                                <fieldset class="fieldsetCSS">
+                                    <legend class="legendCSS">Filters</legend>
```

Comment:
Fieldset now has a Legend that isn't hidden, James gave it tasteful styling as well.  Rules can be viewed in `main.css`

Reply:
![Screen Shot 2024-11-23 at 10 27 35 PM](https://github.com/user-attachments/assets/02020be6-8ec3-406e-941e-cd305f31a966)